v5_MIGRATION_GUIDE.md

```diff
@@ -335,9 +345,9 @@ class MyActivity : FragmentActivity() {
 
 
     fun initializeClients() {
-        braintreClient = BraintreeClient(context, "TOKENIZATION_KEY_OR_CLIENT_TOKEN")
+-       braintreClient = BraintreeClient(context, "TOKENIZATION_KEY_OR_CLIENT_TOKEN")
 -       localPaymentClient = LocalPaymentClient(this, braintreeClient)
-+       localPaymentClient = LocalPaymentClient(braintreeClient)
++       localPaymentClient = LocalPaymentClient(ClientParams(context, "TOKENIZATION_KEY_OR_CLIENT_TOKEN")
 -       localPaymentClient.setListener(this)
```

Comment:
I like this! It's a change we can make on iOS too (in a major version), dropping the merchant step to instantiate a `BTAPIClient`. It always felt redundant to me to have merchants instantiate that class when there was no public functionality on it they would ever use.

Reply:
It'd be great to coordinate this major version alongside the `package` access modifier in Swift 5.9+ ([proposal here](https://github.com/apple/swift-evolution/blob/main/proposals/0386-package-access-modifier.md)) as a while. But totally agree having merchant just access the client like this would be dope:
```swift
// iOS v6
let apiClient = BTAPIClient(authorization: "AUTHORIZATION")
let cardClient = BTCardClient(apiClient: apiClient)

// iOS v7
let cardClient = BTCardClient(authorization:"AUTHORIZATION")
```
Added this to our v7 doc!

Reply:
OMG if `package` is approved that will benefit us in so many ways! 🤞